### PR TITLE
More config cleanup

### PR DESCRIPTION
**Pull request by bdunne**
- Drop balancers
- mod_ssl is already loaded
- ErrorLog is a duplicate

### CODE REVIEW

File: docker-assets/manageiq.conf [hunk 1]
```diff
@@ -1,9 +1,6 @@
 # Timeout: The number of seconds before receives and sends time out.
 Timeout 120
 
-# HTTP Start-up error log
-ErrorLog /var/log/httpd/miq_apache.log
```
Comment (carbonin) [inline docker-assets/manageiq.conf:17]: What about this isn't working?

Also, if we let the errors go to `/var/log` they won't be on the PV.

Comment (carbonin) [reply docker-assets/manageiq.conf:17]: Never mind, I was thinking this was the httpd running on our appliance. This is Fine.

Comment (bdunne) [reply docker-assets/manageiq.conf:17]: The file isn't even created, but there is an error_log in the directory already so no need to duplicate it.

There is no PV for this currently.

**Review by carbonin — APPROVED** [review]

File: docker-assets/manageiq.conf [hunk 2]
```diff
@@ -45,21 +30,21 @@ RewriteRule ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R]
   ProxyPass /cws/ http://${MANAGEIQ_SERVICE_NAME}:9002/cws/
 
   ### API redirects
-  ProxyPass /api balancer://evmcluster_ws/api
-  ProxyPassReverse /api balancer://evmcluster_ws/api
+  ProxyPass /api https://${MANAGEIQ_SERVICE_NAME}
```
Comment (carbonin) [inline docker-assets/manageiq.conf:33]: @bdunne Found the login issue. Must be that we're missing the `/api` here

File: docker-assets/manageiq.conf [hunk 3]
```diff
@@ -45,21 +30,21 @@ RewriteRule ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R]
...
   RewriteCond %{DOCUMENT_ROOT}/%{REQUEST_FILENAME} !-f
-  RewriteRule ^/ balancer://evmcluster_ui%{REQUEST_URI} [P,QSA,L]
-  ProxyPassReverse / balancer://evmcluster_ui/
+  RewriteRule ^/ https://${MANAGEIQ_SERVICE_NAME}%{REQUEST_URI} [P,QSA,L]
+  ProxyPassReverse / https://${MANAGEIQ_SERVICE_NAME}/
 
   ### WebSocket redirects
-  ProxyPass /ws balancer://evmcluster_websocket/ws
-  ProxyPassReverse /ws balancer://evmcluster_websocket/ws
+  ProxyPass /ws ws://${MANAGEIQ_SERVICE_NAME}
```
Comment (carbonin) [inline docker-assets/manageiq.conf:46]: Same here

File: docker-assets/manageiq.conf (hunk 2 above)
Comment (bdunne) [reply docker-assets/manageiq.conf:33]: Right, but this didn't exist before either, so not sure how it worked.

Comment (carbonin) [reply docker-assets/manageiq.conf:33]: I'm not sure what you mean ... we replaced `balancer://evmcluster_ws/api` with `https://${MANAGEIQ_SERVICE_NAME}`

It looks like the replacement should have been `https://${MANAGEIQ_SERVICE_NAME}/api`, right?

**Pull request merged**